.gitlab/new-e2e_testing/windows.yml

```
@@ -48,3 +62,14 @@ new-e2e-windows-agent-msi-windows-server-a7-x86_64:
   rules:
     - !reference [.on_default_new-e2e_tests_a7]
     - !reference [.on_windows_installer_changes_or_manual]
+
+new-e2e-windows-agent-domain-tests-a7-x86_64:
+  stage: kitchen_testing
+  extends:
+    - .new_e2e_template
+    - .new-e2e_windows_domain_test
+    - .new-e2e_windows_a7_x86_64
+    - .new-e2e_agent_a7
```

My PR [modified](https://github.com/DataDog/datadog-agent/blob/main/.gitlab/new-e2e_testing/windows.yml#L47-L54) these so I don't think it will work as is anymore. `.new-e2e_windows_a7_x86_64` now includes/extends `.new-e2e_agent_a7` and `.new-e2e_windows_msi`.

You might be able to just change the order
```suggestion
    - .new-e2e_windows_a7_x86_64
    - .new-e2e_windows_domain_test
    variables:
      EXTRA_PARAMS: ""
```

but it might be better to take out `.new-e2e_windows_msi` and/or move the `TARGETS` and `EXTRA_PARMS` options to a new `.new-e2e_windows_msi_test`